e2e/package.json

```diff
@@ -1,36 +1,37 @@
 {
-    "name": "e2e-scripts",
-    "version": "1.0.0",
-    "description": "E2E test scripts for PWA Kit",
-    "main": "index.js",
-    "scripts": {
-        "test": "pwa-kit-dev test",
-        "test:watch": "pwa-kit-dev test --watch",
-        "test:coverage": "pwa-kit-dev test --coverage",
-        "lint": "eslint \"**/*.js\"",
-        "lint:fix": "eslint \"**/*.js\" --fix"
-    },
-    "devDependencies": {
-        "@actions/core": "^1.11.1",
-        "@aws-sdk/client-s3": "^3.450.0",
-        "@aws-sdk/client-sts": "^3.450.0",
-        "@salesforce/pwa-kit-dev": "3.12.0-dev",
-        "jest": "^26.6.3"
-    },
-    "jest": {
-        "setupFilesAfterEnv": [
-            "./jest.setup.js"
-        ],
-        "testMatch": [
-            "<rootDir>/scripts/**/*.test.js"
-        ],
-        "testPathIgnorePatterns": [
-            "<rootDir>/tests/",
-            "<rootDir>/node_modules/",
-            "<rootDir>/build/"
-        ]
-    },
-    "dependencies": {
-        "dotenv": "^17.2.1"
-    }
+  "name": "e2e-scripts",
+  "version": "3.12.0-dev.1",
```

Comment:
Bonus: for ease of maintenance, aligning this e2e package version with other packages.